spec/models/dialog_field_check_box_spec.rb

```diff
@@ -19,6 +19,41 @@
     end
   end
 
+  describe "#initialize_value_context" do
+    context "dynamic" do
+      let(:dialog_field) { described_class.new(:dynamic => true) }
+      before do
+        allow(DynamicDialogFieldValueProcessor).to receive(:values_from_automate).with(dialog_field).and_return("f")
+      end
+
+      it "returns automate vals" do
+        dialog_field.initialize_value_context
+
+        expect(dialog_field.value).to be_truthy
```

Comment:
I know this has to do with our wonky "t" or "f", but it's kind of confusing that we're expecting a `be_truthy` for an "f". Technically, that's correct, but it's also correct if the default value is a "t" (as shown in the next spec down).

I think both of those specs should probably be changed from `be_truthy` to what the value is really supposed to be.